adam/ontology/phase1_ontology.py

```diff
@@ -434,22 +546,270 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
             bigger_than(_BIRD_SCHEMA_TORSO, _BIRD_SCHEMA_HEAD),
             bigger_than(_BIRD_SCHEMA_TORSO, _BIRD_LEGS),
             supports(_BIRD_LEGS, _BIRD_SCHEMA_TORSO),
-            supports(_BIRD_SCHEMA_TORSO, _BIRD_SCHEMA_HEAD),
-            supports(_BIRD_SCHEMA_TORSO, _BIRD_SCHEMA_TAIL),
-            supports(_BIRD_SCHEMA_TORSO, _BIRD_WINGS),
+            supports(
+                _BIRD_SCHEMA_TORSO,
+                [
+                    _BIRD_SCHEMA_HEAD,
+                    _BIRD_SCHEMA_TAIL,
+                    _BIRD_SCHEMA_LEFT_WING,
+                    _BIRD_SCHEMA_RIGHT_WING,
+                ],
+            ),
         ]
     ),
 )
 
-_BALL_SCHEMA = ObjectStructuralSchema(BALL)
-_BOX_SCHEMA = ObjectStructuralSchema(BOX)
-_WATER_SCHEMA = ObjectStructuralSchema(WATER)
-_JUICE_SCHEMA = ObjectStructuralSchema(JUICE)
-_BOX_SCHEMA = ObjectStructuralSchema(BOX)
-_MILK_SCHEMA = ObjectStructuralSchema(MILK)
-_DOOR_SCHEMA = ObjectStructuralSchema(DOOR)
-_HAT_SCHEMA = ObjectStructuralSchema(HAT)
-_COOKIE_SCHEMA = ObjectStructuralSchema(COOKIE)
+# schemata describing the hierarchical physical structure of objects
+_HOUSE_SCHEMA_LEFT_WINDOW = SubObject(_WINDOW_SCHEMA)
+_HOUSE_SCHEMA_RIGHT_WINDOW = SubObject(_WINDOW_SCHEMA)
+_HOUSE_SCHEMA_DOOR = SubObject(_DOOR_SCHEMA)
+_HOUSE_SCHEMA_ROOF = SubObject(_ROOF_SCHEMA)
+_HOUSE_SCHEMA_WALL = SubObject(_WALL_SCHEMA)
+
+# House modeled after a simple 1 story home as commonly seen in child's books
+# Stick example below -- ASCII art perhaps isn't the best demonstration form
+#      / \
+#    /     \
+#  /         \
+# /           \
+# -------------
+# | []  _  [] |
+# [----| |----]
+_HOUSE_SCHEMA = ObjectStructuralSchema(
+    HOUSE,
+    sub_objects=[
+        _HOUSE_SCHEMA_DOOR,
```

Comment:
Omit doors and windows until we have support for surfaces. For now just have two sub-objects - maybe `GROUND_FLOOR` and `ROOF`